src/components/Card.jsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from 'gatsby'
+import React from 'react'
+
+const Card = ({
+  eyebrowText,
+  mainHeading,
+  content,
+  linkUrl,
+  linkText,
+  imgUrl,
+  imageDescription,
+}) => {
+  return (
+    <section>
+      <div className="card-image-container">
+        <img src={imgUrl} alt={imageDescription} />
```

Comment:
Let's use [Gatsby Image](https://www.gatsbyjs.com/plugins/gatsby-plugin-image#dynamic-images) for this. Shouldn't change the markup that much. Just import `GatsbyImage` and then change the props from `src` to `image`. And of course the `img` tag to a `GatsbyImage` tag and we should be good.